labs.yml

```
@@ -46,3 +46,12 @@ commands:
       - name: report
         default: all
         description: Report type
+  - name: setup-recon-secrets
```

```suggestion
  - name: configure-secrets
```
This makes it more generic and helps us expand this command if needed

---

Done